src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function complete(result) {
+  const div = document.createElement('div');
+
+  div.innerHTML = 'Promise was resolved!';
```

Comment:
```suggestion
  div.textContent= 'Promise was resolved!';
```

Reply:
Use right methods. You are passing just text, it's not html, so you don't need to use `innerHTML` :)